### DISCUSSION TITLE
Toggle ui does not show correctly with Mysql tinyint db value.

### DISCUSSION THREAD

**Discussion by simonpau** [directus/directus#13443] [Archive]
In MySQL 5.7, when the field is tinyint and use toggle ui as interface, edit a collection record, it will not show "checked" in checkbox even tinyint value is 1.

### To Reproduce

Just make a collection with db have a tinyint field
new a record and set the tinyint field to 1 in db
set using toggle ui for that field
edit this record.

**Comment by azrikahar** [comment]
Just curious, is this a fresh instance directly on `9.10.0`, or was this upgraded to `9.10.0` (could be gradually as well)?

Would you mind trying the recommendation here: https://github.com/directus/directus/issues/8480#issuecomment-1083162881? Which means the steps would be:

- verify the `special` column of the field in `directus_fields` is `cast-boolean`
- if it's not, do change it to `cast-boolean` and let us know that it wasn't correctly migrated to `cast-boolean`
